beacon-chain/monitor/process_attestation.go

```diff
@@ -198,6 +198,13 @@ func (s *Service) processAggregatedAttestation(ctx context.Context, att *ethpb.A
 	if s.trackedIndex(att.AggregatorIndex) {
 		log.WithFields(logrus.Fields{
 			"ValidatorIndex": att.AggregatorIndex,
```

Comment:
Unrelated to this PR, could we change it to? 👇 

```suggestion
			"AggregatorIndex": att.AggregatorIndex,
```

Reply:
Done, thanks!